acos_client/v30/slb/virtual_port.py

```diff
@@ -142,9 +140,9 @@ def _set(
         if source_nat_pool and len(source_nat_pool) > 0:
             params['port']['pool'] = source_nat_pool
         if tcp_template:
-            params['port']['tcp_template'] = tcp_template
+            params['port']['template-tcp'] = tcp_template
```

Comment:
@hthompson-a10 
This code seems to override `L123`. Should we allow it? or more conditions needs to be added for preference.